net-libs/gwenhywfar/gwenhywfar-5.10.2.recipe

```diff
@@ -0,0 +1,96 @@
+SUMMARY="Multi-platform helper library for security applications and libraries"
+DESCRIPTION="OS abstraction functions for various projects"
+HOMEPAGE="http://www.aqbanking.de"
+COPYRIGHT="Martin Preuss"
+LICENSE="GNU LGPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/aqbanking/gwenhywfar/archive/refs/tags/$portVersion.tar.gz"
+CHECKSUM_SHA256="2b0b9dd49b95f025f4e0c0346ba88e20b893407c444fccf6403a4da349954f04"
+
+ARCHITECTURES="all"
```

Comment:
This doesn't work with gcc2 (at least because of GTK 3), so for 32 bit x86, it will need to use `SECONDARY_ARCHITECTURES="x86"`, and here

```suggestion
ARCHITECTURES="all !x86_gcc2"
```